api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -3520,4 +3528,78 @@ public void getConcepts_shouldPassWithAndOrNotWords() {
 		assertEquals(1, searchResults.size());
 		assertThat(searchResults.get(0).getWord(), is("SALBUTAMOL INHALER NOT"));
 	}
+	
+	/**
+	 * @see ConceptService#getConceptByReference(String)
+	 */
+	@Test
+	public void getConceptByReference_shouldFindAConceptByItsConceptName() {
+		String conceptName = "COUGH SYRUP";
+		assertEquals("3", conceptService.getConceptByReference(conceptName).getConceptId().toString());
+	}
+	
+	/**
+	 * @see ConceptService#getConceptByReference(String)
+	 */
+	@Test
+	public void getConceptByReference_shouldFindAConceptByItsConceptId() {
+		String conceptId = "3";
```

Comment:
Is this a constant?

Reply:
The constant is an int. Wanted to bring out that the ID is of type int.

Reply:
Can you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions